config/layout.yml

```diff
@@ -28,24 +37,42 @@ Layout/AccessModifierIndentation:
   Enabled: true
   EnforcedStyle: outdent
 
+# Our predominant style of writing multiline arrays is:
+#
+# my_array = [
+#   element_1,
+#   element_2,
+# ]
+#
+# even_in_a_method_call([
+#   element_1,
+#   element_2,
+# ])
+#
 # Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
```

Comment:
Seems like this should be changed if we're changing the rule

Reply:
Sure. I've removed it, as the original purpose was to track the commit messages, which (sometimes) had (more) explanation in them.